### DISCUSSION TITLE
How to use the Class Compile Transformer without a prefix (:uno:)

### DISCUSSION THREAD

**Discussion by alex3025** [unocss/unocss#3596] [Q&A]
Hello, as the title says I would like to hash all of my classes (using [the compile class transformer](https://unocss.dev/transformers/compile-class)) without having to put a prefix in every single class attribute.

Is that possibile?

**Comment by zyyv** [comment]
You can write a vite plugin add the transformer-compile-class trigger tag before unocss utilities in class attribute.

**Reply by alex3025** [reply]
I'm using Nuxt 3

**Comment by 0x241F31** [comment]
There is `trigger` option:
https://github.com/unocss/unocss/blob/main/packages/transformer-compile-class/src/index.ts#L34

Would like to know if anyone has succesfully used it to parse expressions like
```
class=""
class={""}
class={cx``}
```
